examples/clusters/aws-swarm-asg.yml

```diff
@@ -229,6 +229,27 @@ Parameters:
     Default: 8080
     MinValue: 1025
     MaxValue: 32768
+  NoLogs:
```

Comment:
I would prefer a more generic approach, that won't require changing the template and rebuild AMI next time we add a sub stack (I have a branch with a new sub stack for node management). A parameter such as "ApplicationEnv" for instance, in which we'll have something like "NO_METRICS=true REGISTRATION=true NOTIFICATIONS=email". We'll have to parse it in the userdata, but at least it would be generic.

Reply:
While I don't disagree fundamentally, I think this can be part of a separate PR